.github/workflows/ci.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           - "'++2.11.12 test'"
           - "'++2.12.13 test' scripted"
           - "'++2.13.4 test'"
-          - "'++3.0.0-M2 test'"
```

Comment:
munit dropped support, so we can still support M2 for Metals, but not give any guarantees